core/src/main/scala/akka/persistence/jdbc/journal/JdbcAsyncWriteJournal.scala

```diff
@@ -40,11 +41,14 @@ object JdbcAsyncWriteJournal {
 }
 
 class JdbcAsyncWriteJournal(config: Config) extends AsyncWriteJournal {
+  
   implicit val ec: ExecutionContext = context.dispatcher
   implicit val system: ActorSystem = context.system
   implicit val mat: Materializer = ActorMaterializer()
   val journalConfig = new JournalConfig(config)
 
+  PluginVersionChecker.check()
```

Comment:
The most popular plugin component is the write journal. We don't need to check when loading the snapshot component because it's unlikely that someone will be using the snapshot without the journal. That would even not work.